app/test_split_integer.py

```diff
@@ -2,20 +2,20 @@
 
 
 def test_sum_of_the_parts_should_be_equal_to_value() -> None:
-    pass
+    assert sum(split_integer(17, 4)) == 17
```

Comment:
It's great that you're testing whether the sum of the parts is equal to the original number. However, the test could be more robust. You're only testing with the numbers 17 and 4. It would be better to test with a range of numbers to make sure your function can handle different inputs.